Modules/Quest/AvailableQuests/AvailableQuests.lua

```diff
@@ -274,31 +316,54 @@ end
 
 ---@param starter table Either an object or an NPC
 ---@param quest Quest
----@param tooltipKey string the tooltip key. For objects it's "o_<ID>", for NPCs it's "m_<ID>"
-_AddStarter = function(starter, quest, tooltipKey)
+---@param tooltipKey string the tooltip key. For objects it's "o_<ID>", for NPCs it's "m_<ID>", for items it's "im_<ID>" or "io_<ID".
+---@return number added The amount of notes that were added (excluding waypoints)
```

Comment:
Is this some sort of emmy trick to add the variable name (`added` in this case)? Otherwise I suggest:

```suggestion
---@return number The amount of notes that were added (excluding waypoints)
```